frontend/src/components/staking/PageValidatorCharts/index.vue

```diff
@@ -117,15 +120,22 @@ export default {
     tooltips
   }),
   computed: {
+    ...mapState({
+      isNetworkFetching: state => state.connection.isNetworkFetching
+    }),
+    ...mapState({ network: state => state.connection.network }),
+    ...mapState({ networks: state => state.connection.networks }),
     ...mapState([`connection`]),
-    networkId() {
-      return this.connection.networkConfig.id
-    },
     eventsHistory() {
       return this.allHistory.length ? generateEventHistory(this.allHistory) : {}
     }
   },
   watch: {
+    isNetworkFetching: function() {
+      const chainTitle = this.$route.params.chaintitle
+      const network = this.networks.find(net => net.chain_title === chainTitle)
```

Comment:
And if not find? network set as undefined - it's broken all pages. You need to use default value in this case